static/app/views/releases/detail/utils.tsx

```diff
@@ -31,24 +31,23 @@ export type CommitsByRepository = {
  */
 export function getFilesByRepository(fileList: CommitFile[]) {
   return fileList.reduce<FilesByRepository>((filesByRepository, file) => {
-    const {filename, repoName, author, type} = file;
```

Comment:
@JonasBa any reason you are removing this destructuring?

Reply:
for clarity right?

Reply:
Yeah, I find it easier to read w/o local variables. I tend to prefer local variables if there is some method chaining like
```ts
Promise().then(mapdata).then(reduceData)
```
in that case using variables can make it more readable and easier to debug/log what's happening at each step.
```ts
const data = await Promise()
const mappedData = data.map()
const reducedData = data.reducer()
```

Other than that, I think the local variables like the ones I removed are very low value and I'd rather just have prettier format this nicely for me.